dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -255,12 +257,13 @@ void WebdatasetLoader::ReadSample(vector<Tensor<CPUBackend>>& sample) {
     current_wds_shard->Seek(component.offset);
 
     // Skipping cached samples
-    const std::string source_info =
+    const std::string image_key =
```

Comment:
```suggestion
    const std::string sample_key =
```
We should gradually phase out the word `image` since we no longer process just images.

Reply:
Changed